bigtable/tests/unit/test_table.py

```diff
@@ -362,7 +362,8 @@ def _read_row_helper(self, chunks, expected_result):
         request_pb = object()  # Returned by our mock.
         mock_created = []
 
-        def mock_create_row_request(table_name, row_key, filter_):
+        def mock_create_row_request(table_name, row_key, filter_,
+                                    app_profile_id=None):
             mock_created.append((table_name, row_key, filter_))
```

Comment:
This mock should append the `app_profile_id` to the tuple (after `filter_`), and then the code below (line 390) should assert that it is there, and matches the value passed by the caller.  At least one of the testcases (e.g., `test_read_row_complete` at line 399) should pass through a non-None value for the argument.